Assets/Scripts/DogAI/DogAI.cs

```diff
@@ -0,0 +1,116 @@
+﻿/*
+ * Author(s): Timothy Ng
+ * Description: Brains behind the dogs actions on the world
+ * Usage: [no notes]
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using k = PPGlobal;
+
+public class DogAI : MonoBehaviour {
+
+    enum State
```

Comment:
I would expect we'll want to be able to check a dog's state elsewhere though. And potentially will also be used other places in future.

The general policy for enums in the codebase is that they're public and not owned by a class